src/test/java/core/basesyntax/FruitShopTest.java

```diff
@@ -0,0 +1,65 @@
+package core.basesyntax;
+
+import core.basesyntax.dao.RecordDao;
+import core.basesyntax.dao.RecordDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.services.Report;
+import core.basesyntax.services.ReportImpl;
+import core.basesyntax.services.actions.ActionHandler;
+import core.basesyntax.services.actions.BalanceHandler;
+import core.basesyntax.services.actions.IncreaseHandler;
+import core.basesyntax.services.actions.PurchaseHandler;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class FruitShopTest {
```

Comment:
we can remove this test - there is no need to test the class with `main()` method  - please see checklist

we should better test our services independently